queries.yaml

```diff
@@ -502,6 +502,13 @@ post_process:
     params:
       source_layer: roads
       start_zoom: 0
+      end_zoom: 7
```

Comment:
This section isn't needed because we don't source from OpenStreetMap at the low zooms.